recipes-kernel/linux/linux-mainline-rt/defconfig

```diff
@@ -221,9 +222,9 @@ CONFIG_IGBVF=y
 # CONFIG_NET_VENDOR_MARVELL is not set
 # CONFIG_NET_VENDOR_MICREL is not set
 # CONFIG_NET_VENDOR_MICROSEMI is not set
+# CONFIG_NET_VENDOR_NI is not set
```

Comment:
why this change ?

Reply:
I don't know, I've regenerated the defconfig. I guess it is because something has changed in the kernel or the defconfig was not generated correctly. Anyway, it has no impact at all at the end.